eosdis-nasa/earthdata-pub-api
# Edpub 1073 implement new email template

# Description

Implements new email templates and improves send consistency

## Spec

See Ticket: [EDPUB-1073](https://bugs.earthdata.nasa.gov/browse/EDPUB-1073)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Deploy branch to sit.
3. Make sure your account is a data producer and daac staff
4. Run a full workflow to completion and ensure all emails are received.  
5. Add an additional user to a conversation and ensure a DM email is sent.

---

## Change Log

Implements new email templates and improves send consistency

* [Fix _____](link to commitid)
